js/packages/openinference-core/src/utils/index.ts

```diff
@@ -7,12 +7,17 @@ export * from "./typeUtils";
  * @param fn - A function to wrap with a try-catch block.
  * @returns A function that returns null if an error is thrown.
  */
-export function withSafety<T extends GenericFunction>(fn: T): SafeFunction<T> {
+export function withSafety<T extends GenericFunction>(
+  fn: T,
+  fallbackMessage?: string,
```

Comment:
I would make this an object that has an on error callback. That keeps this generic and scale forward. FallbackMessage as a positional argument just makes this function too specific

Reply:
I guess it could be an on error calllback